internal/provisioner/nginx.go

```diff
@@ -87,6 +87,9 @@ func (n *nginx) Destroy() error {
 	return nil
 }
 
+func (n *nginx) ValuesPath() string {
+	return model.UtilityValuesDirectory() + "/nginx_values.yaml"
+}
```

Comment:
nit: looks like we may want an empty line inserted after this line before the next method.

Reply:
Doesn't show as Outdated because it's the next line, but I added a line here while handling the merge https://github.com/mattermost/mattermost-cloud/pull/320/commits/70372c2e80b542e7a577eedae7e774d594d3c47e